python/scrabble-score/scrabble_score.py

```diff
@@ -1,23 +1,19 @@
 def score(word):
 
-    word = word.lower()
```

Comment:
Calling lower is such a simple operation that it should be inlined as we're not using that word elsewhere anyway...